addOns/client/src/main/java/org/zaproxy/addon/client/ExtensionClientIntegration.java

```diff
@@ -140,6 +141,8 @@ public class ExtensionClientIntegration extends ExtensionAdaptor {
     private ClientSpiderDialog spiderDialog;
     private ZapMenuItem menuItemCustomScan;
 
+    private ScanStatus scanStatus = null;
```

Comment:
- Should be clear it's for passive given that the extension has several types of scans.
- The null initialisation is not needed.